dash-renderer/src/components/core/DocumentTitle.react.js

```diff
@@ -5,14 +5,16 @@ import PropTypes from 'prop-types';
 class DocumentTitle extends Component {
     constructor(props) {
         super(props);
+        const {update_title} = props.config;
         this.state = {
             initialTitle: document.title,
+            update_title: update_title,
```

Comment:
I was not sure whether to use camel- or snake-case here. If you prefer camel-case I can change it right-away.

Reply:
Snake is good. Then this can in fact be simplified:
```suggestion
            update_title,
```